sb/robot/sim_robot.py

```diff
@@ -137,10 +278,30 @@ def __init__(self, simulator):
                                    type=pypybox2d.body.Body.DYNAMIC)
             self._body.create_polygon_fixture([(-half_width, -half_width),
                                                (half_width, -half_width),
-                                               (half_width,  half_width),
-                                               (-half_width,  half_width)],
+                                               (half_width, half_width),
+                                               (-half_width, half_width)],
                                               density=500 * 0.12)  # MDF @ 12cm thickness
         simulator.arena.objects.append(self)
+        self.motor_board = MotorBoard(self)
+        self.servo_board = ServoBoard(self)
+        self.camera = Camera(self)
+
+    def __str__(self):
+        return "Robot"
+
+    __repr__ = __str__
+
+    @property
+    def motor_boards(self):
```

Comment:
This won't work, because we use a custom dictionary for this. 

You also seem to be missing the single-index properties

Reply:
I've copied in the `BoardList` class in and done it properly at https://github.com/sourcebots/simulator/pull/1/commits/cef843ef34383f6db7d48174f9f220bbb170b8ff